<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgShowColumns</class>
 <widget class="QDialog" name="DlgShowColumns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chGroup">
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chUnit">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chBox">
     <property name="text">
      <string>Box</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chDiscount">
     <property name="text">
      <string>Discount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chEmarks">
     <property name="text">
      <string>Emarks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chCheckDiscount">
     <property name="text">
      <string>Check discount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chCheckDiscount_2">
     <property name="text">
      <string>Learn accumulate goods</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
